File: BACKEND/src/main/java/com/macro/macro/Model/Usuario.java
package com.macro.macro.Model;


import com.google.cloud.Timestamp;
import lombok.Getter;
import lombok.Setter;




@Getter
@Setter
public class Usuario {

    private String id;
    private String nome;
    private String email;
    private String SenhaHash;
    private String perfil;
    private String status;
    private Timestamp createdAt;
    private Timestamp updatedAt;


}

File: FRONTEND/MC/src/app/model/User/usuario.ts
export interface Usuario {
  id: string;
  FotoPerfil: string;
  nome: string;
  email: string;
  senhaHash: string;
  perfil: string;
  status: string;
  createdAt: string;
  updatedAt: string;
}

export interface UsuarioDTO {
    
  FotoPerfil: string;
  nome: string;
  email: string;
  senha: string;
  perfil: string;
  status: string;


}

export interface UpdateUsuarioDTO {
  fotoPerfil: string;
  nome: string;
  email: string;
  perfil: string;
  status: string;
  
}

export interface UpdateSenhaDTO {
  senhaAtual: string;
  novaSenha: string;

}
